src/realm/array.cpp

```diff
@@ -1995,11 +1789,12 @@ void Array::verify() const
     REALM_ASSERT(m_width == 0 || m_width == 1 || m_width == 2 || m_width == 4 || m_width == 8 || m_width == 16 ||
                  m_width == 32 || m_width == 64);
 
-    if (!m_parent)
+    auto parent = get_parent();
```

Comment:
I believe the general consensus in the team is to not use `auto` for simple types (in this case, `parent` will always be of type `ArrayParent*`, correct?).